### PR TITLE
Uses the newest available distro on first package or repo page load

### LINKED ISSUE #97: Repository page should default to newest available distro
For example, the repository's shortest URL:

https://index.ros.org/r/geometry

Displays Indigo by default, which is equivalent to:

https://index.ros.org/r/geometry/#indigo

What do you think of showing the newest version instead?

### PR DESCRIPTION
**Pull request by hidmic**
Closes #97 by ensuring that, on first page load, when no preferred ros distro has been previously selected, we default to newest available distro for that package or repo.

### CODE REVIEW

File: js/distro_switch.js [hunk 1]
```diff
@@ -57,12 +55,12 @@ $(document).ready(function(){
   }
 
   if(typeof distro == 'undefined') {
-    distro = "{{site.distros[0]}}";
+    distro = default_distro;
```
Comment (apojomovsky) [inline js/distro_switch.js]: @hidmic some wild trailing whitespaces here

**Review by apojomovsky — APPROVED** [review]
Just a single very minor comment, but this LGTM!

File: js/distro_switch.js (hunk 1 above)
Comment (hidmic) [reply js/distro_switch.js]: Plus some spurious newlines. I need a better linter integration with Emacs. Fixed.

**Comment by hidmic** [comment]
Ok, let's get this one in.

**Pull request merged**